pom.xml

```diff
@@ -307,7 +307,7 @@ limitations under the License.
     <connection>scm:git:https://github.com/codehaus-plexus/plexus-pom.git</connection>
     <developerConnection>${project.scm.connection}</developerConnection>
     <tag>master</tag>
-    <url>https://github.com/codehaus-plexus/plexus-pom/tree/master/</url>
+    <url>https://github.com/codehaus-plexus/plexus-pom/tree/${project.scm.tag}/</url>
```

Comment:
This one will point edit icon to read only tag ... why not pointing to master branch?

Also scm report will point to tag ... I would like prefer a master here

Reply:
scm report for a release goes to the release, isn't it?

Reply:
notice that this part of the PR is independant from everything else: I can remove it from this PR and we have a separate discussion on this aspect

Reply:
All other POMs (Maven) have this. I care for consistency.